draft-packages/button/KaizenDraft/Button/Button.tsx

```diff
@@ -1,13 +1,18 @@
 import React, { forwardRef, Ref } from "react"
+import { logComponent } from "@kaizen/react-deprecate-warning/src/util"
 import GenericButton, {
   ButtonProps,
   ButtonRef,
 } from "./components/GenericButton"
 
 const Button = forwardRef(
-  (props: ButtonProps, ref: Ref<ButtonRef | undefined>) => (
-    <GenericButton {...props} ref={ref} />
-  )
+  (props: ButtonProps, ref: Ref<ButtonRef | undefined>) => {
+    logComponent(
```

Comment:
The `forwardRef` prevented us from using `withDeprecatedComponent` so we implemented the logger directly instead